tensorflow/core/platform/default/build_config.bzl

```diff
@@ -8,7 +8,7 @@ load("//tensorflow:tensorflow.bzl", "if_not_mobile")
 WITH_GCP_SUPPORT = False
 WITH_HDFS_SUPPORT = False
 WITH_XLA_SUPPORT = False
-WITH_JEMALLOC = True
+WITH_JEMALLOC = False
```

Comment:
I need to re-test that, I do remember reading code where when one would use jemalloc it would force some things that looked to be problematic for us.